feat(AriaLive): add new component (#3217)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/components/aria-live/info.mdx

---
showTabs: true
---

## Description

AriaLive is a React component and hook that helps make your web app more accessible by announcing dynamic changes to screen readers.

Use it to automatically inform users using a screen reader, about changes on the screen that they didn't initiate.

By default, the `AriaLive` component will announce changes to the screen reader in a polite manner. This means that the announcement will be made when the screen reader is idle. This is the recommended way to use the component.

## Usage

For text content:

```tsx
import { AriaLive } from '@dnb/eufemia'
render(<AriaLive>unvisible message to announce</AriaLive>)
```

> For content that is not visible, but should be announced on changes:

Author: Ture! I think this would be better:

> For content that is visible, but where changes need to be announced:

> For content that is not visible, but should be announced on changes:

Author: ```suggestion
For content that is visible, but where changes need to be announced:
```

True! I think this would be better.
